src/pods/canvas/canvas-put-table-on-top.business.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { putTableOnTop } from './canvas.business';
+import { TableVm } from './canvas.vm';
+
+describe('putTabbleOnTop', () => {
+  it('should return empty array, if I have a table and the array is empty', () => {
+    // Arrange
+    const tables: TableVm[] = [];
+    const tableId = 'someTableId';
+
+    // Act
+    const result = putTableOnTop(tableId, tables);
+
+    // Assert
+    expect(result).toEqual([]);
+  });
+
+  it('should return the same array, if table and array with one element,but it is not the same as table', () => {
+    // Arrange
+    const tables: TableVm[] = [
+      { id: '1', fields: [], tableName: 'Table 1', x: 0, y: 0 },
+    ];
+    const tableId = '2';
+
+    // Act
+    const result = putTableOnTop(tableId, tables);
+
+    // Assert
+    expect(result).toEqual(tables);
+  });
+
+  it("should return an array with a single table, which is the same table if it's the only one", () => {
+    // Arrange
+    const TableId = '1';
+    const tables: TableVm = {
+      id: TableId,
+      fields: [],
+      tableName: 'Table 1',
+      x: 0,
+      y: 0,
+    };
+    const arrayWithOnlyTable: TableVm[] = [tables];
+
+    // Act
+    const result = putTableOnTop(TableId, arrayWithOnlyTable);
+
+    // Assert
+    expect(result).toEqual(arrayWithOnlyTable);
+  });
+
+  it("should return array with table on top if it's already in the first position", () => {
```

Comment:
When we put a table on top we move it to the last position in the array (to ensure that is drawn the last one).

I think the description of this test is not accurate, what about?

```diff
it("should return array with table on top if it's already in the first position", () => {
+ it(`Should return and array with the selected table in the last position of the table arrays`)
```